llvm/include/llvm/IR/CallingConv.h

```diff
@@ -222,6 +222,14 @@ namespace CallingConv {
     // Calling convention between AArch64 Advanced SIMD functions
     AArch64_VectorCall = 97,
 
+    /// Calling convention between AArch64 SVE functions
+    AArch64_SVE_VectorCall = 98,
```

Comment:
I suppose this value came from an earlier revision? No problem, it's harmless, I'm just wondering.

Reply:
Yes, that’s right. There were a couple merge conflicts that I tried to handle in a reasonable way. The other was in a test.

Reply:
Ack, thanks.